app/views/users/index.html.erb

```diff
@@ -0,0 +1,17 @@
+<main>
+    <section>
+      <ul>
+        <% @users.each do |user|%>
+          <%= link_to user do %>
+            <li class="card hover">
+              <div class="user-image border"></div>
+              <div class="user-data border">
+                <h1><%= user.name %></h1>
+                <p class="number-posts">Number of posts: <%= user.posts_counter %></p>
+              </div>
+            </li>
+          <% end %>
+        <% end %>
```

Comment:
- [ ] Dear @Cisco-the-wayword, You did well displaying the `user.name` and `user.posts_counter` of the user, But as the requirement mentioned we need to display the `user.photo` alongside these details. Please add a display for the user photo.
- [Optional] Consider using partials to render the user. create a file in `app/views/users/_user.html.erb` and add the details for displaying a single user in there, once that is done you can reuse this partial for displaying the user details.

Reply:
There is literally a display for the user photo right there

Reply:
Dear @Cisco-the-wayword, Please be reminded he ` <div class="user-image border"></div>` is not doing anything at this point. Please make use of the `user.photo` as a source and display the picture.